Packages/Netherlands3D/Core/Runtime/Scripts/TileSystem/SubObjects.cs

```diff
@@ -156,11 +156,6 @@ private IEnumerator LoadMetaData(Mesh mesh)
 
         var metaDataName = mesh.name.Replace(".bin", "-data.bin");
 
-#if !UNITY_EDITOR && UNITY_WEBGL
-		if(brotliExtention.Length>0)
-			metaDataName += brotliExtention;
```

Comment:
Op dit moment zijn er 3 sets van gegenereerde bestanden, hebben al deze sets dit niet nodig? Of breekt er een van die sets na deze change?

Reply:
die hebben ze wel, maar ik heb de manier waarop .br toegevoegd wordt veranderd zodat dat in een build in de bron url wordt aangepast, zo hoef je niet in elk los script dat een bestand wil hebben deze preprocesser code toe te voegen, en deze was ik vergeten weg te halen. met deze code krijg je [tile.bin.br.br](http://tile.bin.br.br/)

Reply:
Perfect!